### ISSUE TITLE
MacOS : snap remove microk8s not supported - How to remove

### ISSUE THREAD

**Issue by didier-durand** [ubuntu/microk8s#1672]
Hello,

I installed microk8s 1.19 via snap on MacOs

When trying to remove, it says:

*>snap remove microk8s
Interacting with snapd is not yet supported on darwin.
This command has been left available for documentation purposes only.*

How should I remove microk8s properly to reinstall it fresh ?

Thanks

Didier

**Comment by ktsakalozos** [comment]
Hi,

Could you point me to the docs/instructions you followed to get snapd installed on darwin?

Thanks

**Comment by didier-durand** [comment]
Hi there,
Good point: I used https://ubuntu.com/tutorials/install-microk8s-on-mac-os#1-overview

Indeed,I used 'brew install' to install on MacOs. So,now, after your kind reminder, 'brew uninstall' went fine.  Maybe worth mentionning at the end of the tutorial above.

In fact, I confused with native Ubuntu side where I also install microk8s via snap. Sorry, for the noise!

I close this ticket. Thanks!

Didier
